mail-mta/postfix/files/postfix.service

```diff
@@ -12,8 +12,15 @@ ExecReload=/usr/sbin/postfix reload
 PrivateTmp=yes
 PrivateDevices=yes
 ProtectSystem=full
+ReadWritePaths=-/etc/mail/aliases.db
 CapabilityBoundingSet=~ CAP_NET_ADMIN CAP_SYS_ADMIN CAP_SYS_BOOT CAP_SYS_MODULE
 MemoryDenyWriteExecute=true
+ProtectKernelModules=true
```

Comment:
This option was introduced with systemd-232. How do older versions (especially currently stable 226-r2) behave if you use this service file?